Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:14-alpine
+
+RUN apk update
+RUN apk upgrade
+
+WORKDIR /app
+
+ADD . ./
+
+RUN npm install
+
+EXPOSE 3000
+
+RUN npm run build
+
+CMD [ "npm", "run", "preview"]
```

Comment:
I assume that Docker support would be to run a production build of this, not for development. [The documentation for `svelte-kit preview`](https://kit.svelte.dev/docs#command-line-interface-svelte-kit-preview) says that it should be only used for testing, especially with the `--host` option.

Since the project uses `@sveltejs/adapter-node`, would it be possible to run the output of that instead? [The adapter's readme](https://github.com/sveltejs/kit/tree/master/packages/adapter-node#readme) suggests that it should be just a matter of running `node build`.